DATA/SHIPS/loadouts.ini

```diff
@@ -2897,9 +2897,9 @@ equip = DockingLightRedSmall, HpDockLight02
 
 [Loadout]
 nickname = MSN04_King
-archetype = li_elite
-equip = ge_le_engine_01
-equip = shield01_mark02_hf, HpShield01
+archetype = li_fighter
+equip = ge_lf_engine_01
+equip = shield01_mark02_lf, HpShield01
```

Comment:
I don't really like this change. The shooting script explicitly mentions that the group of approaching ships (including King) are flying Liberty Heavy Fighters (Defenders).

```
Suddenly, a new group appears – coming in on a high vector.
One Cruiser and six fighters!
When these ships get within 1200m of the Unity their reputation changes from red to green.
It’s Walker, King and five other LSF ships (flying Liberty Heavy Fighters!).
```